lib/version_utils.pm

```diff
@@ -119,7 +119,8 @@ sub is_storage_ng {
 }
 
 sub is_sle12_hdd_in_upgrade {
-    return get_var('UPGRADE') && !sle_version_at_least('15', version_variable => 'HDDVERSION');
+    return (get_var('UPGRADE') || get_var('ONLINE_MIGRATION') || get_var('ZDUP') || get_var('AUTOUPGRADE'))
```

Comment:
It is duplicate with `is_upgrade_test`

Reply:
Hwww, I put duplicated code here because I thought it was weird to "use main_common 'is_upgrade_test';" in vertion_utils.pm